frontend/public/components/utils/timestamp.jsx

```diff
@@ -128,13 +128,11 @@ export class Timestamp extends SafetyFirst {
       return timestamp;
     }
 
-    return <div>
-      <i className="fa fa-globe" />
-      <div className="co-timestamp">
-        <Tooltip content={[<span className="co-nowrap" key="nowrap">{ mdate.toISOString() }</span>]}>
-          { timestamp }
-        </Tooltip>
-      </div>
+    return <div className="co-timestamp">
+      <i className="fa fa-globe co-timestamp__icon" />
```

Comment:
Not new, but I've been trying to add aria-hidden="true" when I change an icon and see it missing.